app/services/case_create_service.rb

```diff
@@ -71,7 +70,7 @@ def set_flagged_for_disclosure_errors
 
   def flag_for_disclosure_if_required
     @case.save!
-    @flash_notice = "#{@case.type_abbreviation} case created<br/>Case number: #{@case.number}".html_safe
+    @flash_notice = "#{@case.decorate.type_abbreviation} case created<br/>Case number: #{@case.number}".html_safe
```

Comment:
We dont need to use decorate if we move the type abbreviation into the model itself